### PR TITLE
Corrections to Blob.

### PR DESCRIPTION
**Pull request by jpmedley**
Additions to #4581

### CODE REVIEW

File: api/Blob.json [hunk 1]
```diff
@@ -209,7 +209,7 @@
               }
             ],
             "chrome_android": {
-              "version_added": true
+              "version_added": "25"
```
Comment (vinyldarkscratch) [inline api/Blob.json]: This all looks great, just have one little question.  Chrome Desktop had the `webkit` prefix from version 5 to version 25.  Technically, wouldn't that mean Chrome Android 18 would have the `webkit` prefix as well?

Comment (jpmedley) [reply api/Blob.json]: Fixed. Please look again.

Comment (vinyldarkscratch) [reply api/Blob.json]: Thanks for clarification!

File: api/Blob.json [hunk 2]
```diff
@@ -208,9 +208,16 @@
...
             ],
-            "chrome_android": {
-              "version_added": true
-            },
+            "chrome_android": [
+              {
+                "version_added": "25"
+              },
+              {
+                "version_added": "5",
```
Comment (vinyldarkscratch) [inline api/Blob.json]: Whoops, looks like this got set as “5” when it should be “18”, correct?

Comment (jpmedley) [reply api/Blob.json]: Sorry. Fixed.

Comment (vinyldarkscratch) [reply api/Blob.json]: All good, thanks for the quick fixes! This pull request looks all ready to go!

**Pull request merged**